### PR TITLE
[TF FE] Add support for TensorScatterAdd in TF FE

### PR DESCRIPTION
**Pull request by yuhuishi-convect**
### Details:
Add support for [TensorScatterAdd](https://www.tensorflow.org/api_docs/python/tf/raw_ops/TensorScatterAdd)

Test: 
```
cd tests/layer_tests/tensorflow_tests/^C
pytest test_tf_TensorScatterAdd.py 
```

Addresses issue: #25050

### CODE REVIEW

File: src/frontends/tensorflow/src/op_table.cpp [hunk 1]
```diff
@@ -414,6 +414,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops() {
         {"TensorListResize", CreatorFunction(translate_tensor_list_resize_op)},
         {"TensorListConcatV2", CreatorFunction(translate_tensor_list_concat_v2_op)},
         {"TensorScatterUpdate", CreatorFunction(translate_tensor_scatter_update_op)},
+        {"TensorScatterAdd", CreatorFunction(translate_tensor_scatter_add_op)},
```
Comment (rkazants) [inline src/frontends/tensorflow/src/op_table.cpp:417]:
```suggestion
        {"TensorScatterAdd", CreatorFunction(translate_tensor_scatter_add_op)},
        {"TensorScatterUpdate", CreatorFunction(translate_tensor_scatter_update_op)},
```

File: src/frontends/tensorflow_common/src/op/tensor_scater_add.cpp [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#include "common_op_table.hpp"
```
Comment (rkazants) [inline src/frontends/tensorflow_common/src/op/tensor_scater_add.cpp:1]: please insert copyright like in other sources

File: tests/layer_tests/tensorflow_tests/test_tf_TensorScatterAdd.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
+import numpy as np
```
Comment (rkazants) [inline tests/layer_tests/tensorflow_tests/test_tf_TensorScatterAdd.py:1]: copyright

**Comment by yuhuishi-convect** [comment]
@rkazants thanks for the suggestions. Have incorporated the suggested fixes.

**Comment by mlukasze** [comment]
@rkazants to be reviewed

**Comment by hub-bla** [comment]
Hi @yuhuishi-convect, please change `NO` to `YES` in the line below. Without it CI can't run some tests.

https://github.com/openvinotoolkit/openvino/blob/538374f94d73a3dcd2393bfb0b0b11a9e2130472/src/frontends/tensorflow/docs/supported_ops.md?plain=1#L1010

**Comment by p-wysocki** [comment]
Hello @yuhuishi-convect, thanks for your contribution! Is there something we could help you with at the time? The PR seems almost ready :)

**Comment by mlukasze** [comment]
build_jenkins

**Comment by rkazants** [comment]
build_jenkins

**Comment by rkazants** [comment]
> Hi @yuhuishi-convect, please change `NO` to `YES` in the line below. Without it CI can't run some tests.
> 
> https://github.com/openvinotoolkit/openvino/blob/538374f94d73a3dcd2393bfb0b0b11a9e2130472/src/frontends/tensorflow/docs/supported_ops.md?plain=1#L1010

Hi @yuhuishi-convect, any update on this? Also please correct code-style

Best regards,
Roman

**Pull request closed without merging**